src/lib/Microsoft.Health.Common/Telemetry/Exceptions/CustomerLoggedFormattableException.cs

```diff
@@ -0,0 +1,40 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Common.Extension;
+
+namespace Microsoft.Health.Common.Telemetry.Exceptions
+{
+    public class CustomerLoggedFormattableException : IomtTelemetryFormattableException
+    {
+        public CustomerLoggedFormattableException()
+        {
+            this.SetLogToCustomer(true);
+        }
+
+        public CustomerLoggedFormattableException(string message)
+            : base(message)
+        {
+            this.SetLogToCustomer(true);
+        }
+
+        public CustomerLoggedFormattableException(string message, Exception innerException)
+            : base(message, innerException)
+        {
+            this.SetLogToCustomer(true);
+        }
+
+        public CustomerLoggedFormattableException(
+            string message,
```

Comment:
Sorry for getting back late. Apparently, what the customer sees cannot be dynamic (you can't unpack a property bag dynamically). It can contain dynamic columns, but log analytics recommends against it. 
I'm not sure if I'm supposed to post internal documentation here, but I can send you the reference